scripts/publish.sh

```diff
@@ -118,6 +118,20 @@ publish_to_docker_hub() {
 
 }
 
+publish_to_public_ecr() {
+	aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/aws-observability
+	aws ecr-public create-repository --repository-name aws-for-fluent-bit --catalog-data file://awsforfluentbit.json --region us-east-1  || true
```

Comment:
I don't think this is needed, the repo should already exist

Reply:
Yeah.. Make sense. Will remove this line.

Reply:
Ahh kind of confused? Can't we test before merging the code into mainline?